cityofaustin/atd-moped
# Adds milestone progress column to dashboard

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/9277
fixes https://github.com/cityofaustin/atd-data-tech/issues/9349

## Testing
**URL to test:** 
https://deploy-preview-702--atd-moped-main.netlify.app/

**Steps to test:**

Go to dashboard, follow and add yourself to the team of a project if there is no project displayed under either tab
Add a project milestone to one of the projects and set it to incomplete, revisit dashboard
Add another project milestone to one of the projects and set it to complete, revisit dashboard and see the milestone column change

Follow a project
Add yourself as a team member of a project
Go to the dashboad
Click betweeb the "following" and "my projects" tabs and note that the table does not rerender when you switch tabs

---
#### Ship list
